components/script/dom/element.rs

```diff
@@ -2575,6 +2616,28 @@ impl Element {
         self.set_click_in_progress(false);
     }
 
+    pub fn get_lang(&self) -> Atom {
+        if let Some(ref lang) = *self.cached_lang.read() {
+            lang.clone()
```

Comment:
Not sure, but as I said, you can't just call `get_attribute` in a method that get's called for layout, that's plain racy.